nxdrive/metrics/poll_metrics.py

```diff
@@ -0,0 +1,69 @@
+from logging import getLogger
+from queue import Empty, Queue
+from time import monotonic_ns
+from typing import TYPE_CHECKING
+
+from ..constants import TIMEOUT
+from ..engine.workers import PollWorker
+from ..exceptions import ThreadInterrupt
+from ..objects import Metrics
+from ..options import Options
+from ..qt.imports import pyqtSlot
+from .constants import ENDPOINT, SYNC_ACTION, SYNC_TIME
+
+log = getLogger(__name__)
+
+
+if TYPE_CHECKING:
+    from ..client.remote_client import Remote
+
+_CHECK_INTERVAL = 900  # 15 minutes
```

Comment:
Move it to constants, just after the endpoint.

Reply:
You can also set it to `60 * 15` without comment, it will be then easy to understand.